File: Modul 2/Bahasa Python/PRAK205-2210817110014-Muhammad Firas.py
import math

angka = [0,1]
angka = list(map(int, input("").split()))

a = int(angka[0])
b = int(angka[1])
c_alas = math.sqrt((b**2)-(a**2))
keliling = a+b+c_alas
luas = 0.5*c_alas*a

print("Alas = %d cm" %c_alas)
print("Tinggi = %d cm" %a)
print("Keliling = %d cm" %keliling)
print("Luas = %d cm^2" %luas)

File: Modul 2/Bahasa C/PRAK205-2210817110014-Muhammad Firas.c
#include<stdio.h>
#include<math.h>

int main() {
    int A, B;
    scanf("%d %d", &A, &B);

    int alas = sqrt((B*B)-(A*A));
    int tinggi = A;
    int keliling = A+B+alas;
    int luas = 0.5*A*alas;

    printf("Alas = %d cm\n", alas);
    printf("Tinggi = %d cm\n", tinggi);
    printf("Keliling = %d cm\n", keliling);
    printf("Luas = %d cm^2", luas);

}